css/main.css

```diff
@@ -23,137 +24,168 @@ body {
 }
 
-h2 {
-    font-size: 1.5rem;
-    margin: 8px 0;
+header h1 {
+    color: #154273;
+    font-size: 28pt;
+    width: 40%;
+    letter-spacing: 1px;
 }
 
-.video {
-    width: 80%;
-    padding-top: 45%;
-    position: relative;
+header h1 .lead {
+    display: block;
+    font-weight: normal;
+    font-size: 0.9em;
 }
 
-.video iframe {
+header img {
+    display: block;
+    top: 30%;
     position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    border: 0;
-}
-
-@media only screen and (max-width: 800px) {
-    article {
-        max-width: 90%;
-    }
+    left: 62.5%;
+    width: 25%;
 }
 
-/* Statements */
-
-details {
+/* Article */
+article {
     position: relative;
-    border: 1px solid #888;
 
-    margin: 32px 0;
-    -moz-appearance:initial;
-    border-right-width: 10px;
+    margin-top: -50px;
+    margin-left: calc(8% - 32px);
+    width: 50%;
+    padding: 0 32px 32px;
+    background-color: white;
+    border-radius: 32px;
 }
 
-details > summary {
-    list-style-type: none;
+article > h2 {
+    padding-top: 25px;
+    margin-bottom: 30px;
 }
 
-details > summary::-webkit-details-marker {
-    display: none;
+article p {
+    width: 70%;
+    line-height: 1.5em;
 }
 
-details > * {
-    margin: 0 48px;
+/* Statements */
+
+.statements {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-left: -16px;
+    margin-right: -16px;
 }
 
-summary {
-    padding-top: 16px;
+.statements details {
+    margin: 16px;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2);
+    flex-grow: 1;
     padding-bottom: 16px;
-    cursor: pointer;
-}
 
-summary:focus {
-    outline: none;
+    background-repeat: no-repeat;
+    background-position: right 15px top 20px;
+    background-size: 15px;
 }
 
-summary::before {
-    content: '+';
-    display: block;
-    background-color: #01689b;
-    width: 20px;
-    height: 20px;
-    position: absolute;
-    top: 28px;
-    left: 16px;
-    font-size: 20px;
-    font-weight: bold;
-    line-height: 20px;
-    text-align: center;
-    border-radius: 10px;
-    color: white;
+.statements details.checked {
+    background-image: url('../img/trafficlight-green.svg');
 }
 
-details[open] summary::before {
-    content: '-';
+.statements details.in-progress {
+    background-image: url('../img/trafficlight-orange.svg');
 }
 
-details[open] {
-    padding-bottom: 32px;
+.statements details.stopped {
+    background-image: url('../img/trafficlight-red.svg');
 }
 
-details.failed {
-    border-right-color: #bb0000;
+.statements details > summary {
+    padding: 24px;
+    list-style-type: none;
+    cursor: pointer;
+    margin-left: 0;
 }
 
+.statements details > * {
+    margin: 0 24px;
+}
 
-details.in-progress {
-    border-right-color: #ff9900;
+.statements details > summary::-webkit-details-marker {
+    display: none;
 }
 
-details.checked {
-    border-right-color: #00bb33;
+.statements summary:focus {
+    outline: none;
```

Comment:
Was al opgemerkt in de Slack; maar in kader van toegankelijkheid en ontdekbaarheid willen we waarschijnlijk wel wat meer stijl-hints dat se `<summary>` kan openklappen

Reply:
Excellent point, I’ve removed the offending CSS to improve accessibility.
A chevron or + symbol should probably also improve things, but I’ll leave that to someone who can do some actual graphic design for now.